manifest.toml

```
@@ -0,0 +1,56 @@
+packaging_format = 2
+
+id = "collabora"
+name = "Collabora Online"
+description.en = "LibreOffice-based online office suite with collaborative editing"
+description.fr = "Suite office en ligne et collaborative, basée sur LibreOffice"
+
+version = "22.05.8.2~ynh2"
+
+maintainers = ["rafi59"]
+
+[upstream]
+license = "MPL-2.0"
+website = "https://collaboraoffice.com"
+admindoc = "https://www.collaboraoffice.com/code/"
+
+[integration]
+yunohost = ">= 11.1.15"
+architectures = ["amd64", "arm64"]
+multi_instance = false
+ldap = "not_relevant"
+sso = "not_relevant"
+disk = "50M"
+ram.build = "50M"
+ram.runtime = "50M"
+
+[install]
+    [install.domain]
+    type = "domain"
+    full_domain = true
+
+    [install.password]
+    type = "password"
+
+    [install.nextcloud_domain]
+    ask.en = "The domain of your Nextcloud instance"
+    ask.fr = "Le domain de votre instance Nextcloud"
+    type = "string"
+    example = "example.com/nextcloud"
+    default = ""
+
+[resources]
+
+    [resources.ports]
+
+    [resources.system_user]
+
+    [resources.install_dir]
+
+    [resources.permissions]
+    main.url = "/"
+
+    [resources.apt]
+    packages = "coolwsd, code-brand, hunspell-de-de, hunspell-en-gb, hunspell-fr, hunspell"
+    extras.repo = "deb https://collaboraoffice.com/repos/CollaboraOnline/CODE-debian11 ./ "
+    extras.key = "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x0C54D189F4BA284D 2>/dev/null"
```

I think it should be something like this : 

```suggestion
    [resources.apt.extras.collabora]
    packages = "coolwsd, code-brand, hunspell-de-de, hunspell-en-gb, hunspell-fr, hunspell"
    repo = "deb https://collaboraoffice.com/repos/CollaboraOnline/CODE-debian11 ./ "
    key = "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x0C54D189F4BA284D"
```